ContactDetailsApi.Tests/V2/E2ETests/Stories/FetchAllContactDetailsTests.cs

```diff
@@ -47,25 +47,25 @@ protected virtual void Dispose(bool disposing)
         }
 
 
-        [Fact]
-        public void ServiceReturnsAllContactDetailsAsRequested()
-        {
-            this.Given(g => _contactDetailsFixture.GivenAFetchAllContactDetailsByUprnRequest())
-                .When(w => _steps.WhenAllContactDetailsAreRequested())
-                .Then(t => _steps.ThenAllContactDetailsAreReturned(_dbFixture))
-                .BDDfy();
-        }
+        //[Fact]
+        //public void ServiceReturnsAllContactDetailsAsRequested()
+        //{
+        //    this.Given(g => _contactDetailsFixture.GivenAFetchAllContactDetailsByUprnRequest())
+        //        .When(w => _steps.WhenAllContactDetailsAreRequested())
+        //        .Then(t => _steps.ThenAllContactDetailsAreReturned(_dbFixture))
+        //        .BDDfy();
+        //}
 
-        [Fact]
-        public void ServiceReturnsUnauthorizedWhenUserIsNotInAllowedGroups()
-        {
-            Environment.SetEnvironmentVariable("AUTH_ALLOWED_GROUPS_EXTERNAL", "unauthorized-group");
+        //[Fact]
+        //public void ServiceReturnsUnauthorizedWhenUserIsNotInAllowedGroups()
+        //{
+        //    Environment.SetEnvironmentVariable("AUTH_ALLOWED_GROUPS_EXTERNAL", "unauthorized-group");
```

Comment:
These are the tests that were introduced during the initial PR for this endpoint (EP)?
I'm guessing they're not needed right now as this EP isn't utilizing Google Groups auth.